utils/github-hook/hook.py

```diff
@@ -168,6 +248,7 @@ def event_processor_dispatcher(headers, body, inserter):
             inserter.insert_event_into(DB, 'issues', result)
         elif headers['X-Github-Event'] == 'pull_request':
             result = process_pull_request_event(body)
+            label_pull_request_event(body)
             inserter.insert_event_into(DB, 'pull_requests', result)
```

Comment:
probably it makes sense to swap these lines, so we don't lose entry in db in case there's some issue with setting labels